src/extensions/default/CodeFolding/main.js

```diff
@@ -54,7 +45,11 @@ define(function (require, exports, module) {
         EXPAND                  = "codefolding.expand",
         EXPAND_ALL              = "codefolding.expand.all",
         gutterName              = "CodeMirror-foldgutter",
-        COLLAPSE_CUSTOM_REGIONS = "codefolding.collapse.customregions";
+        codeFoldingMenuDivider  = "codefolding.divider",
+        collapseKey             = "Ctrl-Alt-[",
+        expandKey               = "Ctrl-Alt-]",
+        collapseAllKey          = "Alt-1",
+        expandAllKey            = "Shift-Alt-1";
```

Comment:
Ack, scratch that -- I had the syntax backwards.  It _does_ work on master.  We should just make sure it still works in this PR too with the new, more dynamic way bindings are registered & unregistered.